### ISSUE TITLE
Obtaining the Collections of Children property nothing

### ISSUE THREAD

**Issue by ehsanh06** [discordjs/discord.js#4982]
**Issue**
I have a Category Channel which has 20+ Text channels from which are a children of this Category channel. When trying to log out the collection using `.array().length` it is returning nothing. I can verify that each other category channels I have in my server, they all return the same thing. 

**index.js**
```js
// Initialise King-bot
client.once('ready', () => {
    console.log('Bzzzztttt... Bot online');

    // Find and store the channels/categories for later use.
    kingVsChannel = findChannel(client,'King vs');
});

// Allow for King-bot to process messages when called 
client.on('message', message => {

    // Destructure param 'message' for readability.    
    const { content, author, channel } = message;

    // Ignore if a message isn't from the '!' or if the Bot itself was replying to its own command.
    // Otherwise process the command/args that the user has input and reply with a message in the channel.
    if (!content.startsWith(prefix) || author.bot) {
        return;
    }

    // Args is essentially the word(s) we enter after we write the command with prefix.
    const args = content.slice(prefix.length).split(/ +/);

    const command = args.shift().toLowerCase();
    console.log( kingVsChannel.children.array().length );
    console.log( kingVsChannel.children; )
});
```

**utils.js**
```
/**
 * Dynamically finds the channel we specify.
 * @param {*} client - the instance object of the Discord bot.
 * @param {*} channelName - the channel name we want want to return.
 * @param {*} channelType - must be either 'text', 'guild', 'category', or 'voice'.
 */
const findChannel = (client, channelName) => client.channels.cache.find(channel => channel.name === channelName);
```

**Outputs for each Console log**
_.children.array().length_
```
console.log( kingVsChannel.children.array().length );
Outputs: 0
```

_.children_
```
console.log( kingVsChannel.children );
// Outputs: Collection [Map] {}
```

**NOTE**
I've checked the Bot permissions in this particular Category and it has read/write access on the category channel, I've checked this vs other bots I've got in the server and there's no real differences.

**Further details:**

- discord.js version: 12.4.0
- Node.js / NPM version: v12.14.0 / v6.14.8
- Operating system: Windows 10 Pro
- Priority this issue should have – please be realistic and elaborate if possible:  Low priority - this Bot is a side-project so it's absolutely fine.

**Relevant client options:**

- partials: none
- gateway intents: none
- other: none

- [ ] I have also tested the issue on latest master, commit hash:

**Comment by ehsanh06** [comment]
Which 'latest master' are you referring to?

**Comment by wasdennnoch** [comment]
There was an issue with mapping channels to their categories in v12.4.0 which has been fixed in v12.4.1. Do you still encounter the issue on the updated version?

**Comment by ehsanh06** [comment]
Literally had no idea they released a patch - just did a fresh `npm install` and I can see the collection coming back, I'll close this issue down. Thanks for the prompt response @wasdennnoch seriously appreciated.
